projects/graduated/riscv-dv.md

```diff
@@ -0,0 +1,40 @@
+# Project application: `riscv-dv`
+
+## Application
+
+### Sandbox application
+
+* Project name: `riscv-dv`
+* Project repo(s): https://github.com/google/riscv-dv
+* Brief summary of the project: `riscv-dv` is an open source verification tool for RISC-V processors. `riscv-dv` is a SystemVerilog based random RISC-V instruction generator that checks the execution against an industry standard ISS for correction and compliance.
+* Project's open source license: Apache 2.0
+* Link to issue tracker: https://github.com/google/riscv-dv/issues
+* Link to website: None.
+* Links to social media accounts: None.
+* Who uses this project, and at what scale: Google, OpenTitan, Western Digital, Andes, and others.
+* Why does the project want to join CHIPS Alliance: To enable the CHIPS Alliance community to design and validate RISC-V CPU Cores.
+* Primary contact from the project during the Sandbox application process:
+ * Name: Matt Cockrell
+ * Email: mcockrell@google.com
+ * GitHub handle: @mcockrell-google
+ * Role within the project: `riscv-dv` Workgroup Lead
+
+*Projects applying for Sandbox status may stop here.*
+
+### Graduation application
+
+* Total number of active committers: https://github.com/google/riscv-dv/graphs/contributors
+* Brief description of release methodology and mechanics: Releases are automated and published on every commit.
```

Comment:
there only seems to be one release? there is some kind of CI but it does not seem to generate a build package. I would propose you rather use the following phrase: "No regular release policy has been adopted as of yet; a policy is going to be developed in collaboration within CHIPS.", unless I am misunderstanding something...

Reply:
@mcockrell-google can you comment?

Reply:
Went and fixed it myself. Please correct the wording I used if it's imprecise / wrong, but just wanted to put this forward for a formal vote.